index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Hunter Woody</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Oswald:400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Boogaloo" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.10/css/all.css" integrity="sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <h1 class="header-name">Hunter Woody</h1>
+            <nav class="navbar">
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="project.html">Projects</a></li>
+                    <li><a href="resume.html">Resumé</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="clearfix">
+
+        </div>
+
+        <section class="bio">
+            <div class="my-photo-contain">
+                <img src="me.jpg" class="my-photo">
+            </div>
+            <div class="text-contain">
+                <h3 class="hello">Hello! I am...</h3>
+                <h1 class="bio-name">Hunter Woody</h1>
+                <h2 class="momentum">Momentum Student</h2>
+                <p class="bio-text">I am a current student in Momentum Learning's first cohort. I'm from Durham, NC and have lived here all my life. Some of my interests include anything tech related and anything sports related.</p>
+            </div>
+            <div class="clearfix"></div>
+        </section>
+
+        <div class="push"></div>
+    </div>
+    <footer class="footer">
+        <a href="mailto:hunterwoody@gmail.com" class="mail"><i class="fas fa-envelope-square fa-3x"></i></a>
+        <a href="https://github.com/hsw919" class="github" target="_blank"><i class="fab fa-github-square fa-3x"></i></a>
```

Comment:
I see you found `target="_blank"`. Great job! I have some thoughts on this -- in general, it's not a great user experience to have a new tab open when you don't expect it -- but that's debatable, and your execution is good.